### ISSUE TITLE
So I will create issues here. Here is first assignment.

### ISSUE THREAD

**Issue by CliqueAdmin** [Mahi451/CirclesConsort#3]
![image](https://user-images.githubusercontent.com/57764284/111790273-4385d300-887f-11eb-88af-3a275617f8f5.jpeg)

Please use navigation you learned in the course and add the portion as highlighted in the above picture. Send me pull request.

Also commit measages are wron. Please follow below guidelines going forward.

https://www.google.com/amp/s/chris.beams.io/posts/git-commit/amp/

Please make sure you do not commit directly. Always send a pull request. 

Goal is not to develop in hurry but to do it in right way and properly.

**Comment by CliqueAdmin** [comment]
@Mahi451 Please let me know once you read it.

**Comment by Mahi451** [comment]
Read what?😬 This https://www.google.com/amp/s/chris.beams.io/posts/git-commit/amp/

**Comment by CliqueAdmin** [comment]
Yes that and also the screenshot I gave you to implement with navigation.
